server/controller/db/mysql/migration/rawsql/issu/6.4.1.0.sql

```diff
@@ -0,0 +1,19 @@
+DROP PROCEDURE IF EXISTS add_chpodgroup_podgrouptype;
+
+CREATE PROCEDURE add_chpodgroup_podgrouptype()
+BEGIN
+    DECLARE pod_group_type_column CHAR(32) DEFAULT '';
+
+    SELECT COLUMN_NAME INTO pod_group_type_column
+    FROM INFORMATION_SCHEMA.COLUMNS
+    WHERE TABLE_SCHEMA = 'deepflow' AND TABLE_NAME = 'ch_pod_group'
+    AND COLUMN_NAME = 'pod_group_type';
+
+    IF pod_group_type_column = '' THEN
+        ALTER TABLE `ch_pod_group` ADD COLUMN `pod_group_type` INTEGER DEFAULT NULL AFTER `name`;
+        ALTER TABLE `vinterface` ADD COLUMN `vmac` CHAR(32) DEFAULT '' AFTER `mac`;
```

Comment:
We have issue executing this query.

There is a same query in `6.3.1.49.sql`
```
ALTER TABLE `vinterface` ADD COLUMN `vmac` CHAR(32) DEFAULT '' AFTER `mac`;
```

When upgrading, this will fail and there is no way to return to the previous version, unless drop the database.

Reply:
I have just completed the modifications
https://github.com/deepflowio/deepflow/pull/4501